src/docs/index.rst

```diff
@@ -39,10 +39,13 @@ The main drivers of the Axom project are to:
 Quickstart Guide
 =====================
 
-The `Axom Quickstart Guide <../../quickstart_guide_docs/html/index.html>`_ 
-contains information about accessing the code, configuring and building, 
-linking with an application, etc.
+The Axom Quickstart Guide contains information about accessing the code,
+configuring and building, linking with an application, etc.
 
+.. toctree::
```

Comment:
Did the link not work? This looked nicer the other way imo.

Reply:
The link did work.  But I wanted the toctree caption, mainly because of the way it shows up in the side bar, and I did not want two links to the quickstart very close to each other.